### ISSUE TITLE
Get gwin_make_workflow to estimate the segment length

### ISSUE THREAD

**Issue by Cyberface** [gwastro/gwin#60]

**Comment by ahnitz** [comment]
@Cyberface I wasn't privy to this discussion, can you add a bit about about this? Is it just having a default? The reason one might change this is if one has a model that is sensitive to line behavior, then one needs to have better line whitening and a correspondingly longer filter.

**Comment by Cyberface** [comment]
@ahnitz this is wrong sorry. I made the issue during the meeting so we wouldn't forget to do it but the title is misleading. It's not to do with the inverse truncation length so I'll change the title.

I think what this is supposed to mean is:

We should have a way for `gwin_make_workflow` to estimate the segment length for the analysis.
Which can be calculated from the prior information. i.e. what is the longest waveform in your prior.
Would we want to round this to the next power of 2?

I'm imagining this as an option that is by default computed by the code but you can override it by simply specifying it in the config file or override.

Does that make more sense?

**Comment by ahnitz** [comment]
@Cyberface Ah, that's much clearer now, thanks!
